### DISCUSSION TITLE
Duplicated styles

### DISCUSSION THREAD

**Discussion by spokospace** [antfu/vite-ssg#255] [General]
Hello, 

what is the reason for duplicated styles? I noticed the styles are in code, and also in a separate file:

![image](https://user-images.githubusercontent.com/16397823/171175500-143e3717-db86-4a5e-ac50-237c2e42652b.png)

![image](https://user-images.githubusercontent.com/16397823/171175651-182dffc3-c429-4745-bd3c-11fc91ab1f6a.png)

**Comment by antfu** [comment]
It's a feature https://github.com/antfu/vite-ssg#critical-css

You can remove `critters` to disbale it
